openfecwebapp/constants.py

```diff
@@ -209,11 +209,10 @@
 ])
 
 candidate_status = OrderedDict([
-    ('C', 'Statutory candidate'),
+    ('C', 'For selected election cycle'),
```

Comment:
Maybe too nit-picky, but I'm not sure this is right. First of all, there isn't always a selected election cycle. Second, the candidate records are using the *most recent* candidate status value, not the candidate status value(s) of the selected cycle(s), if any.